python_module/CMakeLists.txt

```diff
@@ -29,20 +29,32 @@ if(CREATE_PYTHON_MODULE)
     target_link_libraries(${libname} PRIVATE ${MKL_DEF_LIBRARY})
   endif()
 
+  set(PYTHON_INSTALL_SITE_DIR ${CMAKE_INSTALL_PREFIX}/lib/python${PYTHON_VERSION_MAJOR}.${PYTHON_VERSION_MINOR}/site-packages)
+
+  # set relative RPATH
+  if(isSystemDir STREQUAL "-1")
+    set(CMAKE_INSTALL_RPATH_USE_LINK_PATH TRUE)
+    file(RELATIVE_PATH relDir ${PYTHON_INSTALL_SITE_DIR}/sirius
+      ${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR})
+    message("reldir in python: ${relDir}")
```

Comment:
Perhaps this was only needed for debugging?

Reply:
yes :)